packages/ajv-validator/tsconfig.build.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "rootDir": "src",
+    "outDir": "dist",
+    "tsBuildInfoFile": "dist/tsconfig.build.tsbuildinfo",
+    "emitDeclarationOnly": false
+  },
+  "include": ["src/**/*.ts"],
+  "exclude": ["**/*.test.ts"],
```

Comment:
Only include src files in the build, while excluding `.test.ts` files.

The big benefit of this is it prevents test related globals from appearing as valid references in src files, eg; `describe`, `test`, `expect` from `vitest`